src/controllers/userProfileController.js

```diff
@@ -383,47 +370,48 @@ const userProfileController = function (UserProfile) {
       }
 
       const canEditTeamCode =
-        req.body.requestor.role === "Owner" ||
-        req.body.requestor.role === "Administrator" ||
-        req.body.requestor.permissions?.frontPermissions.includes(
-          "editTeamCode",
-        );
+        req.body.requestor.role === 'Owner' ||
+        req.body.requestor.role === 'Administrator' ||
+        req.body.requestor.permissions?.frontPermissions.includes('editTeamCode');
 
       if (!canEditTeamCode && record.teamCode !== req.body.teamCode) {
-        res.status(403).send("You are not authorized to edit team code.");
+        res.status(403).send('You are not authorized to edit team code.');
+        return;
+      }
+
+      if (!canToggleRequestBio && (record.bioPosted !== req.body.bioPosted || 'default')) {
```

Comment:
```suggestion
      if (!canToggleRequestBio && (record.bioPosted !== req.body.bioPosted || record.bioPosted !== 'default')) {
```
Pretty sure `|| 'default'` would always evaluate to True.

Also, if the user has 'requestBio' permission, but not 'putUserProfile', would this let them edit the rest of the user profile?

Reply:
@nathanah do you suggest I create an endpoint for toggling the bio?